testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -537,6 +537,28 @@ public void testMaterializedView()
                         "FROM\n" +
                         "  nation");
 
+        assertQueryFails(
```

Comment:
these aren't really a connector test. Could you add unit test similar to `TestCreateMaterializedViewTask` for each task that you change?